app/javascript/application.js

```diff
@@ -3,56 +3,46 @@ import "@hotwired/turbo-rails"
 import "./controllers"
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { Routes, Route } from 'react-router-dom';
+import { Routes, Route, Link } from 'react-router-dom';
 import { BrowserRouter } from 'react-router-dom';
 import { Provider } from 'react-redux';
 import store from './redux/store'
 import Main from './components/Main';
-import NavBar from './components/NavBar';
 import SectionDetail from './components/Detail';
 import SectionNew from './components/SectionNew';
 import ReservationNew from './components/ReservationNew';
 import ReservationCreate from './components/ReservationCreate';
 import Reservations from './components/Reservations';
 import CreateRestaurantForm from './components/CreateRestaurantForm';
 import Delete from './components/Delete';
-import { FaTwitter } from 'react-icons/fa';
-import { ImFacebook } from 'react-icons/im';
-import { TiSocialGooglePlus } from 'react-icons/ti';
-import { BsVimeo } from 'react-icons/bs';
-import { FaPinterestP } from 'react-icons/fa';
+import RegisterUser from './components/RegisterUser';
+import Login from './components/Login';
 import '../assets/stylesheets/application.css'
 
+function SplashPage() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry (I found 10):

![image](https://github.com/JuanDBta/booking-bites/assets/57421823/a8b79cf5-a593-4729-bf84-97b0bfed9cbc)


That way, your code will meet the latest standards. 💪